src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,57 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.*;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotContainer;
+import frc.robot.commands.IntakeCommand;
+
+public class IntakeSubsystem extends SubsystemBase {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+ 
+  //intake motors
+  Spark intakeMotor1 = new Spark(kIntakeMotorPort1);
+  Spark intakeMotor2 = new Spark(kIntakeMotorPort2);
+
+  public DigitalInput intakeSwitch = new DigitalInput(kIntakeLimitSwitch);
+  
+  public IntakeSubsystem() {
+    intakeMotor2.setInverted(true);
+  }
+  public void intake() {
+  }
+
+  public void initDefaultCommand() {
```

Comment:
Deprecated, see above for where to implement this

Reply:
be more specific as to what above is referencing